src/core/lib/channel/channel_stack.h

```diff
@@ -159,6 +159,11 @@ typedef struct {
 
   /* The name of this filter */
   const char *name;
+
+  /* For filter with the same \a name, determines which one is kept in the
+   * channel stack. Stringly lesser values take precedence (UNIX priority
```

Comment:
I have no idea where that word came from. It's the result of a race condition in my brain. It should be **Strictly**, but I think my muscle memory is more used to writing "string" when a word begins with "stri".